app/helpers/application_helper.rb

```diff
@@ -31,6 +31,12 @@ def notifications_to_display?
     !current_page?(notifications_path) && (authenticated_user_has_notifications? || unauthenticated_user_has_notifications?)
   end
 
+  def sanitise_characters(string)
+    return string unless string
+
+    string.gsub(/[^a-zA-Z0-9\s\-\_\'\&]/, '').tr("'", "’").tr("&", "＆")
+  end
+
```

Comment:
Helper method that:

- Replace apostrophes with the typographic character `’`.
- Replace ampersands with the full-width character `＆`.
- Remove any characters that are not alphanumeric, spaces, hyphens, underscores, apostrophes, or ampersands.